components/App.js

```diff
@@ -30,7 +30,7 @@ class App extends React.Component {
   render() {
     return (
       <>
-        <Overlay onFinish={this.loadConvertKit} />
```

Comment:
do I need to do something to load convert kit if we remove this line? Not sure how that works...

Reply:
Move calling it to the body of `componentDidMount()`. Or just rename `loadConvertKit` to be `componentDidMount`.